examples/combined_ODNP.py

```diff
@@ -283,9 +292,10 @@
         )
         time_axis_coords[j + thermal_scans]["stop_times"] = time.time()
     DNP_data.set_prop("stop_time", time.time())
-    DNP_data.set_prop("postproc_type", "spincore_ODNP_v4")
```

Comment:
shouldn't be hardset

Reply:
It **absolutely should be**. This is how the processing code understands what version of the code this is, allowing it to adapt the preproc.  In fact, you should not hardset this value **in your processing script**!